### PR TITLE
Jax sigmoid

### PR DESCRIPTION
**Pull request by JosephWick**
Use `jax.nn.sigmoid` function

### CODE REVIEW

File: diffsmhm/galhalo_models/utils.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import numpy as np
 from jax import numpy as jax_np
-
+from jax.nn import sigmoid as sigmoid
```
Comment (beckermr) [inline diffsmhm/galhalo_models/utils.py:3]:
```suggestion
from jax.nn import sigmoid as sigmoid

```

File: diffsmhm/galhalo_models/sigmoid_smhm.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 from collections import OrderedDict
 import jax
 from jax import numpy as jax_np
+from jax.nn import sigmoid as sigmoid
```
Comment (beckermr) [inline diffsmhm/galhalo_models/sigmoid_smhm.py:6]:
```suggestion
from jax.nn import sigmoid as sigmoid
```

File: diffsmhm/galhalo_models/utils.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import numpy as np
 from jax import numpy as jax_np
```
Comment (beckermr) [inline diffsmhm/galhalo_models/utils.py:2]:
```suggestion
```
